src/components/form-textarea/form-textarea.js

```diff
@@ -1,19 +1,25 @@
 import Vue from '../../utils/vue'
+import VBVisible from '../../utils/vb-visible'
```

Comment:
Maybe we should move this to a separate `directives` directory like we do with `mixins`.

Reply:
We have a directives dir (for public directives), but only certain directives are published in the docs... so yeah, this could move there maybe.

Reply:
Yeah, that sounds like a good idea.

Reply:
Changed.